### PR TITLE
Readme game instructions

### PR DESCRIPTION
**Pull request by sergiocortessat**
Milestone 4. We have added the proper instructions to the users. Explaining the requirements, and how to run the game in their terminal. As well we provide clear instructions on how to play and the different winning combos there is. The game is complying with the logic requirements and structure organization

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED 🔴 ♻️ 

### Hey @sergiocortessat & @arikarim, 👋🏽 
**This is a great project you have going on, well-done lads**. However, there are a few issues to fix to get approved.

### REQUIRED CHANGES: ⚠️ 

- [x] Kindly fix the spelling and grammatical errors you are having in the read me file.
- [x] Please go ahead and add the descriptive procedures for downloading or cloning to a local computer.
- [x] Your issues link redirects to bubble sort issues link page. You may go ahead and fix that.
- [x] [OPTIONAL] Replace the socials section with your usenames.

- [x] Kindly go through the inline comments below for more required changes.


### Kindly make all the requested changes and submit another CRR. 💯 

**Cheers 🍹 and Happy coding 💻** 

## Reach me on slack for a chit-chat 🆘  Roy Mukuye

File: README.md [hunk 1]
```diff
@@ -33,6 +33,102 @@ Download a copy of the TicTacToe repository
...
 - Follow the game instructions
 
+## Instructions
+
+ Once the game has started
+ 1. First player place the name
+ 2. First player need to select a symbol from X or O
+ 3. Second player needs to enter the name
+ 4. Symbol for second player is assgined automatically
+ 5. Game starts - Makes player1 to chooose a section to place his symbol (1-9)
```
Comment (mke2111) [inline README.md]: The spelling of 'chooose'. Kindly change that.

Always use tools like Grammarly to check your writings

File: README.md [hunk 2]
```diff
@@ -33,6 +33,102 @@ Download a copy of the TicTacToe repository
...
+ 6. If the number you entered is taken, you will be force to chooose again
+ 7. Player 2 will now select an space in the board to place the symbol (same rules apply as step 6)
+ 8. Players need to obtain a winnin combo using their symbol (CHECK BELOW FOR WINNING COMBOS)
+ 9. If none of the players can get a winning combo, the game will simply draw with no winners. 
+ 10. The game will promp you to play once again: press "ok". Or to end the game: press "no"
```
Comment (mke2111) [inline README.md]: The spelling of 'promp'. Kindly change that.

Always use tools like Grammarly to check your writings

File: README.md [hunk 3]
```diff
@@ -33,6 +33,102 @@ Download a copy of the TicTacToe repository
...
+
+ Once the game has started
+ 1. First player place the name
+ 2. First player need to select a symbol from X or O
+ 3. Second player needs to enter the name
+ 4. Symbol for second player is assgined automatically
+ 5. Game starts - Makes player1 to chooose a section to place his symbol (1-9)
+ 6. If the number you entered is taken, you will be force to chooose again
+ 7. Player 2 will now select an space in the board to place the symbol (same rules apply as step 6)
+ 8. Players need to obtain a winnin combo using their symbol (CHECK BELOW FOR WINNING COMBOS)
```
Comment (mke2111) [inline README.md]: The spelling of 'winnin'. Kindly change that.

Always use tools like Grammarly to check your writings

**Comment by sergiocortessat** [comment]
Grammar errors have been fixed within the readme.

**Review by mke2111 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED 🔴 ♻️ 

### Hey @sergiocortessat & @arikarim, 👋🏽 
**Thanks for the changes made**. However, there are a few more issues to fix to get approved.

### REQUIRED CHANGES: ⚠️ 

- [x] Kindly fix the spelling and grammatical errors you are having in the read me file.
- Prerequisites is not correctly spelled. Kindly run you paragraphs through Grammarly to catch all errors.
- [x] Please go ahead and add the descriptive procedures for downloading or cloning to a local computer.
- Add the git cloning command, the run command, and the change directory command.

- [x] Your issues link is still redirecting to bubble sort issues link page. You may go ahead and fix that.
Screenshot from 2021-02-17 12-28-22


- [x] [OPTIONAL] Replace the socials section with your usenames.


### Kindly make all the requested changes and submit another CRR. 💯 

**Cheers 🍹 and Happy coding 💻** 

## Reach me on slack for a chit-chat 🆘  Roy Mukuye

**Comment by sergiocortessat** [comment]
We have added the remaining changes requested. Apologize for not fixing it before requesting the review.

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Changes Required! :recycle:
### Hi @sergiocortessat and @arikarim :clap: :clap:
#### You have done a good job :+1: However, you should fix the contributing section by including the appropriate issue link instead of removing the section entirely. Fix this and submit it for review.

#### _Reviewed by Temitayo Adeleke_

**Review by lekegitrepo — APPROVED** [review]
## Status: Approved :heavy_check_mark:
### Good job @sergiocortessat and @arikarim :100: :1st_place_medal:
#### All the requirements are fixed. This project is approved :white_check_mark:

#### _Reviewed by Temitayo Adeleke_

**Pull request merged**